people/lucia-specia.md

```diff
@@ -0,0 +1,21 @@
+---
+description: Researcher and Professor of natural language processing
+---
+# Lucia Specia
+Lucia Specia is a researcher and Professor of natural language processing. 
```

Comment:
*Professor* lower case

Reply:
For me, this article loses the main point:

That she was a key force for quality estimation research, since the beginning.

> Lucia Specia is a researcher and professor of natural language processing.  She led early research on machine translation [quality estimation](quality-estimation.md).

Concrete bits:

- She organised the QE task at WMT
- She wrote a (the!) book on QE
- She led the research group that released a few early QE libraries